### DISCUSSION TITLE
Changing tailwind styles in an included library don't compile.

### DISCUSSION THREAD

**Discussion by TristanBrotherton** [tailwindlabs/tailwindcss#13576] [Help]
I have a custom svelte UI library that I develop alongside my main project. I have my UI library automatically reloading inside the main project with `svelte-package --watch` So as I make changes my main project will reload them from the package.
Im my main project I have my `tailwind.config.cjs` specify the path to my custom UI library:

```
content: [
        './src/**/*.{html,js,svelte,ts}',
        './node_modules/my custom UI library/**/*.{html,js,svelte,ts}'
    ],
 ```
 
 If I make a change to my custom UI library, i see the classes update in my project straight away. However they will only render if they are included in the project already. If I add a new class that is not already in use it will not be compiled unless I restart my server. I'm at a bit of a loss as to how to fix this so that the classes are rendered in real-time.

**Comment by alex-krasikau** [comment]
Hey @TristanBrotherton!

How do you link the application and the library? Do you use `npm`, `yarn`, or `pnpm link` to integrate your UI library into `node_modules`, or do you have a monorepo configuration?

**Reply by TristanBrotherton** [reply]
Thanks for the reply - I'm using `pnpm link` to integrate the library.

**Reply by alex-krasikau** [reply]
Did I understand correctly that UI library components using unused classes in the main app do not work, but if you include these classes anywhere in the application, they start working?

**Reply by TristanBrotherton** [reply]
Yes that is correct. Unless I restart my main app server, then they will compile.

**Reply by alex-krasikau** [reply]
The setup seems to be correct, and I have no idea what could be causing the issues. Would you be able to create a reproduction for further investigation?
